torch_xla/csrc/ops/tensor_data.cpp

```diff
@@ -0,0 +1,78 @@
+#include "torch_xla/csrc/ops/tensor_data.h"
+
+#include <sstream>
+
+#include "tensorflow/compiler/xla/xla_client/debug_macros.h"
+#include "torch_xla/csrc/lowering_context.h"
+#include "torch_xla/csrc/ops/xla_ops.h"
+#include "torch_xla/csrc/tensor_util.h"
+
+namespace torch_xla {
+namespace ir {
+namespace ops {
+namespace {
+
+template <typename T>
+void EmitValues(std::ostream& stream, const at::Tensor& tensor) {
```

Comment:
I'd replace `Emit` with `Print` for both helpers. `Emit` is an ambiguous term.

Reply:
SG!